source/extensions/common/aws/BUILD

```diff
@@ -31,6 +31,72 @@ envoy_cc_library(
         "//source/common/common:utility_lib",
         "//source/common/crypto:utility_lib",
         "//source/common/http:message_lib",
+    ],
+)
+
+envoy_cc_library(
+    name = "sigv4_signer_impl_lib",
+    srcs = ["sigv4_signer_impl.cc"],
+    hdrs = [
+        "signer_base_impl.h",
+        "sigv4_signer_impl.h",
+    ],
+    deps = [
+        ":credentials_provider_interface",
+        ":signer_interface",
+        ":signer_base_impl",
+        ":utility_lib",
+        # "//source/common/buffer:buffer_lib",
+        # "//source/common/common:hex_lib",
+        # "//source/common/common:logger_lib",
+        # "//source/common/common:utility_lib",
+        # "//source/common/crypto:utility_lib",
+        # "//source/common/http:message_lib",
```

Comment:
If these are unnecessary, please remove these lines.

Reply:
ack

Reply:
Ping please clean this up.

Reply:
ouch - I thought I got these. fixed.